app/views/slides/show.html.erb

```diff
@@ -17,7 +17,7 @@ if @slide.tags.count > 0
    end
 %>
 <% set_meta_tags keywords: keywords %>
-<% set_meta_tags author: user_url(@slide.user_id) %>
+<% set_meta_tags author: "#{root_url}#{@slide.user.username}" %>
```

Comment:
__[rails_best_practices]__
* law of demeter [:ref](http://rails-bestpractices.com/posts/2010/07/24/the-law-of-demeter/)